pkg/internal/apis/config/validate_test.go

```diff
@@ -18,9 +18,10 @@ package config
 
 import (
 	"fmt"
-	"sigs.k8s.io/kind/pkg/internal/assert"
 	"testing"
 
+	"sigs.k8s.io/kind/pkg/internal/assert"
+
```

Comment:
Very trivial nit, but if you do an update for any other reason, looks like the formatting should have grouped these two together and sorted them without the space between.